contrib/grpc-contrib/src/main/java/com/salesforce/grpc/contrib/StaticResolver.java

```diff
@@ -154,8 +154,8 @@ private static class StaticResolverFactory extends NameResolver.Factory {
 
         @Nullable
         @Override
-        public NameResolver newNameResolver(URI targetUri, Attributes params) {
-            return provider.newNameResolver(targetUri, params);
+        public NameResolver newNameResolver(URI uri, NameResolver.Args args) {
```

Comment:
nit: it seems that the first param is still named `targetUri` in grpc::NameResolver (https://github.com/grpc/grpc-java/blame/f082151fb5de1c102a492e2fa2bc4c1bef0d8c6a/api/src/main/java/io/grpc/NameResolver.java#LL151C34-L151C49)
and other places in this PR keep the same naming. Will it be possible to revert this rename?

Reply:
sure